extern/googletest/CMakeLists.txt

```diff
@@ -29,6 +29,11 @@ endif()
 # Silence some warnings with modern versions of CMake on macOS.
 set( CMAKE_MACOSX_RPATH TRUE )
 
+# Respect NaN expressions for IntelLLVM
+if ( "${CMAKE_CXX_COMPILER_ID}" MATCHES "IntelLLVM")
+   detray_add_flag( CMAKE_CXX_FLAGS "-fhonor-nans")
+endif()   
```

Comment:
Let's remove this weak from here as well, for the same reason as stated for GoogleBenchmark.

Reply:
Right